config/config.go

```diff
@@ -339,6 +340,7 @@ var defaultConf = Config{
 	SplitTable:                   true,
 	Lease:                        "45s",
 	TokenLimit:                   1000,
+	StoreLimit:                   -1,
```

Comment:
It's better to set the default value to `0` and make compatible with the old configuration.

Reply:
Good idea.